### ISSUE TITLE
Question on fragment timeouts

### ISSUE THREAD

**Issue by aneebq** [mangui/flashls#469]
In the documentation for `HLSSettings` I noticed that `fragmentLoadMaxRetryTimeout` specifies that there is a one second timeout to try to load the fragment. Is this timeout around how long it takes to download the entire fragment, or does the timeout control the duration to get the first byte of data? Is it possible to tune this parameter so it starts from something other than one second?

https://github.com/mangui/flashls/blob/dev/src/org/mangui/hls/HLSSettings.as#L217

**Comment by aidaima** [comment]
In my opinion one second or fragmentLoadMaxRetryTimeout is only the interval between the twice fragment downloading.

**Comment by mangui** [comment]
indeed,  fragmentLoadMaxRetryTimeout is the interval flashls will wait before retrying loading  after a load error.
I agree the explanation is not that clear :blush:
feel free to post a PR to clarify it
